bin/cdxgen.js

```diff
@@ -104,6 +104,12 @@ const args = yargs(hideBin(process.argv))
   .option("server-url", {
     description: "Dependency track url. Eg: https://deptrack.cyclonedx.io",
   })
+  .option("insecure", {
```

Comment:
Since this flag is only used for dependency track submission, I am wondering if can make it very specific like --insecure-dt-submit or --skip-dt-verification

Reply:
@setchy

Reply:
Yes you are right. The challenge is to find a not too long name for the option ... :-)
What do you think of:

> --skip-dt-tls-check ?

Reply:
lgtm